apps/native-component-list/src/screens/AV/AudioPlayer.tsx

```diff
@@ -1,18 +1,18 @@
 import React from 'react';
 import { StyleProp, ViewStyle } from 'react-native';
-import { AV, Asset } from 'expo';
+import { Asset, Audio } from 'expo';
```

Comment:
I reverted adding the `AV` export to `expo`, introduced by #3975, so we don't need to immediately deprecate it.